lib/yaml.dart

```diff
@@ -30,24 +31,43 @@ export 'src/yaml_node.dart' hide setSpan;
 ///
 /// If [sourceUrl] is passed, it's used as the URL from which the YAML
 /// originated for error reporting.
-dynamic loadYaml(String yaml, {Uri? sourceUrl}) =>
-    loadYamlNode(yaml, sourceUrl: sourceUrl).value;
+///
+/// If [recover] is true, will attempt to recover from parse errors and may return
+/// invalid or synthetic nodes.
+///
+/// If [errorListener] is supplied, its onError method will be called for each
+/// error. If [recover] is false, parsing will end early so only the first error
+/// may be emitted.
```

Comment:
I don't think this paragraph is currently true. There's currently only one thrown exception that's sent to the error listener. Which means that any thrown exception needs to be caught and added to the list of exceptions to report, unless it was already reported to the error listener. I think it would be cleaner if exceptions that are going to be thrown are _not_ reported to the error listener so that clients don't need to remember to make the extra check.

Reply:
Done!